go.mod

```diff
@@ -1,9 +1,11 @@
 module github.com/codecrafters-io/git-tester
 
-go 1.20
+go 1.21
+
+toolchain go1.22.1
```

Comment:
toolchain got added automatically, once I ran `go get -u github.com/codecrafters-io/tester-utils`, had to upgrade CI to go1.21.x to accomodate this. (There was a mismatch in the project go version, and my local toolchain version, we can try to remedy that too.)

Should we keep this ? Or remove it. 
https://go.dev/doc/toolchain

Reply:
This isn't customer facing, so as long as it works we're good